README.md

```diff
@@ -1,71 +1,102 @@
-# Drucker-Parent
+# Rekcurd-Parent
 <img src="./docs/img/logo.png" width="100">
 
-Drucker is a microservice framework for serving a machine learning module. Drucker makes it easy to serve, manage and integrate your ML models into your existing services. Moreover, Drucker can be used on Kubernetes. We strongly recommend to use Kubernetes for a production service.
+Rekcurd is a software package of managing machine learning (ML) modules. Rekcurd makes it "easy to serve ML module", "easy to manage ML model and deploy ML module" and "easy to integrate into the existing service". Rekcurd can be run on Kubernetes.
+
+
+## Features
+- High Availability
+- Free from managing
+  - Upload the latest ML model
+  - Versioning ML models
+  - Evaluate the accuracy of ML model and visualize it (TBD)
+  - Switching ML model to the ML module
+  - Non stop deployment (a.k.a Rolling update) *
+- Free from monitoring
+  - Health check and auto healing *
+  - Load balancing and auto scaling *
+- Others
+  - WebUI available
+  - Service level (development/staging/production) management *
+  - AB testing and canary release (TBD)
+  - Log forwarding *
+  - Easy to integrate into the existing service
+
+*Kubernetes features
+
 
 ## Components
-- [Drucker](https://github.com/drucker/drucker): Serving framework for a machine learning module.
-- [Drucker-dashboard](https://github.com/drucker/drucker-dashboard): Management web service for the machine learning models to the drucker service.
-- [Drucker-client](https://github.com/drucker/drucker-client): SDK for accessing a drucker service.
-- [Drucker-example](https://github.com/drucker/drucker-example): Example of how to use drucker.
+- [Rekcurd](https://github.com/rekcurd/drucker): Project for serving ML module. This is a gRPC micro-framework and it can be used like [Flask](http://flask.pocoo.org/).
+- [Rekcurd-dashboard](https://github.com/rekcurd/drucker-dashboard): Project for managing ML model and deploying ML module. Any Rekcurd service is manageable. It can deploy the Rekcurd service to Kubernetes cluster.
+- [Rekcurd-client](https://github.com/rekcurd/drucker-client): Project for integrating ML module. Any Rekcurd service is connectable. It can connect the Rekcurd service on Kubernetes.
+
+### Sub components
+- [dockerfiles](https://github.com/rekcurd/dockerfiles): Project for Rekcurd's container image. Since it works with [Docker Hub](https://hub.docker.com/r/rekcurd/rekcurd), you can pull image via `docker pull`.
+- [drucker-grpc-proto](https://github.com/rekcurd/drucker-grpc-proto): Project for Rekcurd's gRPC spec.
+- [Drucker-example](https://github.com/rekcurd/drucker-example): Example project. This sample turns a scikit-learn Linear SVC algorithm into a Rekcurd service.
+
```

Comment:
Drucker-example => Rekcurd-example?
drucker-grpc-proto => reckurd-grpc-proto?

If they will be modified when repository name is changed, it is good for now.

Reply:
> If they will be modified when repository name is changed, it is good for now.

Yes we plan to.